web-client/src/views/MyAccount/VerifyNewEmailModal.tsx

```diff
@@ -9,7 +9,7 @@ export const VerifyNewEmailModal = connect(
     cancelSequence: sequences.clearModalSequence,
     closeVerifyEmailModalAndNavigateToMyAccountSequence:
       sequences.closeVerifyEmailModalAndNavigateToMyAccountSequence,
-    email: state.form.email,
```

Comment:
We don't always get the pending email value via a form now since the banner can appear anywhere.